CMakeLists.txt

```diff
@@ -62,81 +61,29 @@ include_directories(${bredis_SOURCE_DIR}/include ${Boost_INCLUDE_DIRS})
 MESSAGE(STATUS "boost inc: " ${Boost_INCLUDE_DIRS})
 MESSAGE(STATUS "boost libs: " ${Boost_LIBRARIES})
 
-add_library(catch_lib t/tests-main.cpp)
+# add_library(catch_lib t/tests-main.cpp)
```

Comment:
I think this line needs to be deleted. because it is not part of CMake file, also it is not a useful comment. By the way there are some lines with same situation in this file.